### ISSUE TITLE
Can't use default when dynamic import for TS module resolution: node16

### ISSUE THREAD

**Issue by jaredhan418** [pinojs/pino-pretty#473]
```
//partial tsconfig
  "compilerOptions": {
    "moduleResolution": "Node16",
    "module": "Node16",
    "target": "ES2022",
  },
```

<img width="389" alt="ts-node16" src="https://github.com/pinojs/pino-pretty/assets/11781874/7dd602cd-9a2c-4c6c-8f40-38edaebde4ca">

TS always throw error cuz default function not as expect when import `pino-pretty` in dynamic way.

**Comment by jaredhan418** [comment]
suggest, add one line `module.exports.PinoPretty= build`  below to match current `index.d.ts`
https://github.com/pinojs/pino-pretty/blob/41fdd31b66506f862ad4ce3ca8a791408522bf8d/index.js#L170

**Comment by mcollina** [comment]
Thanks for reporting! Would you like to send a Pull Request to address this issue?

**Comment by jaredhan418** [comment]
#484 add PR here

**Comment by jaredhan418** [comment]
<img width="364" alt="截屏2023-12-27 16 20 07" src="https://github.com/pinojs/pino-pretty/assets/11781874/16fcbee0-cf35-49ce-8cea-14263fb56802">
Then we can use build function in TS without type error.

**Comment by mcollina** [comment]
Fixed in #484
